package persona.api.account.personal

class InvalidDataException(message: String, cause: Throwable = null) extends RuntimeException(message, cause)
